torchprime/launcher/buildpush.py

```diff
@@ -61,7 +61,8 @@ def buildpush(
     _run(
       f"{sudo_cmd} docker tag {docker_tag} {docker_url}",
     )
-    _run(f"{sudo_cmd} docker push {docker_url}")
+    if torchprime_docker_tag != "local_run":
```

Comment:
This introduces a magical constant. I think it's simpler if we add a "push=True" function argument, and have the other file call this with push=False